pkg/config/config.go

```diff
@@ -255,6 +270,23 @@ func (lci *LocalConfigInfo) DeleteConfiguration(parameter string) error {
 
 }
 
+// DeleteUrl is used to delete config from local odo config
+func (lci *LocalConfigInfo) DeleteUrl(parameter string) error {
+	for elem, i := range *lci.componentSettings.Url {
+		if i.Name == parameter {
+			po := remove(*lci.componentSettings.Url, elem)
+			fmt.Println(po)
+			lci.componentSettings.Url = &po
+		}
+	}
+	return lci.writeToFile()
+
+}
+
+func remove(s []ConfigUrl, i int) []ConfigUrl {
```

Comment:
may be tie this to an alias of []ConfigUrl ? instead of taking it as a parameter and move the lopping logic from DeleteUrl to here?

Reply:
Do we even need this function? It is just one line, and not that complicated one

Reply:
ping @surajnarwade

Reply:
yep, I tried this line directly, it was not working for me, so I kept this function as it is

Reply:
cleared my doubt, removed the `remove` function